storage/internal/test/conformance/retry_tests.json

```diff
@@ -265,7 +268,7 @@
       "description": "downloads_handle_complex_retries",
       "cases": [
         {
-          "instructions": ["return-broken-stream"]
+          "instructions": ["return-broken-stream", "return-broken-stream"]
```

Comment:
I don't see a matching change for this line in https://github.com/googleapis/conformance-tests/pull/78/files

Reply:
Discussed offline, this was another upstream change that the go repo hadn't picked up